src/helpers/createNotificationIdHelper.ts

```diff
@@ -0,0 +1,3 @@
+export const createNotificationIdHelper = () => {
+  return Math.floor(Math.random() * 10000);
```

Comment:
be better get a time in seconds from date

Reply:
I can't use the date because the notification id must be 10 characters

Reply:
then get last 10 numbers from date